pkg/alertmanager/operator.go

```diff
@@ -1919,3 +1920,45 @@ func ApplyConfigurationFromAlertmanager(a *monitoringv1.Alertmanager, updateScal
 
 	return monitoringv1ac.Alertmanager(a.Name, a.Namespace).WithStatus(asac)
 }
+
+func (cb *ConfigBuilder) checkAlertmanagerGlobalConfigResource(
```

Comment:
(nit) since it's now a ConfigBuilder method, it would be good to move it to amcfg.go next to the other methods.